### PR TITLE
Remove un needed style

### PR DESCRIPTION
**Pull request by Noamili**

### CODE REVIEW

**Review by MichalPorag — APPROVED** [review]
Great work! please fix conflicts

**Review by MichalPorag — APPROVED** [review]

File: styles/Home.module.scss [hunk 1]
```diff
@@ -1,27 +1,19 @@
...
+  grid-template-columns: repeat(auto-fit, minmax(350px, 1fr));
+  grid-auto-rows: 1fr;
+  gap: 30px 45px;
+  justify-items: center;
+  margin-bottom: 100px;
+  max-width: 1280px;
+  margin: -24px auto 0 auto;
 }
 
 @media screen and (max-width: 600px) {
```
Comment (MichalPorag) [inline styles/Home.module.scss:14]: please chang to 1280px

File: styles/Home.module.scss [hunk 2]
```diff
@@ -1,27 +1,19 @@
...
-  }
+.cards__wrapper {
+  display: grid;
+  grid-template-columns: repeat(auto-fit, minmax(350px, 1fr));
+  grid-auto-rows: 1fr;
+  gap: 30px 45px;
+  justify-items: center;
+  margin-bottom: 100px;
+  max-width: 1280px;
+  margin: -24px auto 0 auto;
```
Comment (Noamili) [inline styles/Home.module.scss:11]: 24 instead of 0

File: styles/Home.module.scss [hunk 3]
```diff
@@ -1,27 +1,19 @@
...
 
 @media screen and (max-width: 600px) {
-  .container {
-    width: 100%;
-
-    .cards__wrapper {
-      grid-template-columns: repeat(auto-fit, minmax(250px, 1fr));
-    }
+  .cards__wrapper {
+    grid-template-columns: repeat(auto-fit, minmax(250px, 1fr));
```
Comment (MichalPorag) [inline styles/Home.module.scss:16]: please remove tis

File: styles/Home.module.scss [hunk 4]
```diff
@@ -1,27 +1,19 @@
...
-    gap: 30px 45px;
-    justify-items: center;
-    padding: 0 25px;
-    margin: 0 auto;
-    max-width: 1769px;
-    margin-bottom: 100px;
-  }
+.cards__wrapper {
+  display: grid;
+  grid-template-columns: repeat(auto-fit, minmax(350px, 1fr));
```
Comment (MichalPorag) [inline styles/Home.module.scss:5]: Please change to 300px min

**Review by MichalPorag — CHANGES_REQUESTED** [review]
Great work! just a few comments

**Review by MichalPorag — APPROVED** [review]
Geat work!

**Pull request merged**